fe/fe-core/src/main/java/com/starrocks/connector/ConnectorManager.java

```diff
@@ -0,0 +1,37 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.connector;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import javax.annotation.concurrent.GuardedBy;
+
+// ConnectorManager is responsible for managing all ConnectorFactory, and for creating Connector
```

Comment:
and also manage all connector instance

Reply:
CatalogMgr only mange catalog.

Reply:
and catalog holds an instance of connector

Reply:
In my opinion, catalog only use to persist.

Reply:
Isn't catalog a symbol of external data source?

Reply:
In user view, it is. 
but in project, connector is the important role to connect external data source. 
I think it's better to separate connector and catalog. FYI

Reply:
No, there are some info only remain in catalog, such as a rate limit of specific catalog, it's not a good idea to let connector be in charge of it.